code/src/App.js

```diff
@@ -1,9 +1,58 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react';
 
+
+import { API_URL} from "./reusable/urls";
+import ThoughtsForm from "./Components/ThoughtsForm";
+import ThoughtsCard from "./Components/ThoughtsCard";
+import Counter from "./Components/Counter";
+
+//useState variables here in the export function
```

Comment:
Very good job in being able to divide the different functions into many component, makes it easy to find relevant code for each part of the app. 🤩